openid-4-verifiable-presentations-1_0.md

```diff
@@ -1821,6 +1822,90 @@ The following is a non-normative example of the payload of a Self-Issued ID Toke
 
 Note: The `nonce` and `aud` are set to the `nonce` of the request and the Client Identifier of the Verifier, respectively, in the same way as for the Verifier, Verifiable Presentations to prevent replay.
 
+### Conditional Credential Request Flow
+
+When combining OpenID for Verifiable Presentations with [@!SIOPv2], the Verifier has the flexibility to determine whether to request credentials based on the authenticated user's context. The aforementioned flow can be modified as follows:
+
+1. The user initiates the authentication process with the Verifier.
+2. The Verifier starts the [@!SIOPv2] flow and sends an Authentication Request to the user's Wallet, omitting the `presentation_definition` and `presentation_definition_uri` parameters.
+3. The Wallet processes the Authentication Request and performs user authentication using the [@!SIOPv2] mechanism.
+4. Upon successful authentication, the Wallet sends the Authentication Response back to the Verifier, including the `id_token`.
+5. The Verifier validates the `id_token` and extracts the necessary information to identify the user.
+6. Based on the user's identity and the Verifier's context, the Verifier determines whether additional credentials are required.
+   - If no additional credentials are needed, the Verifier proceeds with issuing the authentication token to the user.
```

Comment:
what does `issuing the authentication token to the user` mean..?

Reply:
how would you rephrase this? perhaps `If no additional credentials are needed, the Verifier proceeds`

or what is the expected action after successful presentation?